examples/api/test/widgets/framework/error_widget.0_test.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/widgets/framework/error_widget.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('ErrorWidgetExampleApp', () {
```

Comment:
We don't use group for api example tests because those test files usually contain very few tests.
For this PR, consider removing groups and updating the tests titles (the code in the setUp method can be added at the beginning of each test).